src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/Utils.cs

```diff
@@ -41,5 +42,10 @@ internal static Uri AppendQueryString(Uri url, string qs)
             builder.Query = newQueryString;
             return builder.Uri;
         }
+
+        internal static bool IsRunningBlazorWasm()
+        {
+            return RuntimeInformation.IsOSPlatform(OSPlatform.Create("WEBASSEMBLY"));
```

Comment:
Paging @SteveSandersonMS @danroth27 in case there are any changes to how we should be detecting the Browser environment.

Reply:
We're following up on this. For now, this is the right API to use.